pool/pool.py

```diff
@@ -372,6 +382,10 @@ async def create_payment_loop(self):
                 pool_coin_amount = int(total_amount_claimed * self.pool_fee)
                 amount_to_distribute = total_amount_claimed - pool_coin_amount
 
+                if total_amount_claimed < 1.75:
+                    self.log.info(f"Do not have enough funds to distribute: {total_amount_claimed}, skipping payout")
```

Comment:
This will still distribute any value sent to the wallet.

Is there a way to identify how that coin as generated, using the parent coin info here and associate with the singleton?

Reply:
Yes but that's more complicated, and not necessary in reference

Reply:
Would you mind giving a quick run down on how that would work?

Or is there a section in any doc we can point us to on how that works?

Appreciate it.

Reply:
To add a little more context, usually the parent_coin_info of the rewarded coin equals the singleton_coin_record. But that is not true 100% of the time.

Reply:
if it's complicated to get it right, it would be nice if the reference pool to show how it should be done

Reply:
Reference pool is not meant to be a tutorial / guide, and it's also not meant to be a fully production ready pool.
It works fine as is, so we will not be adding in more optional /  nice to have features.
However if you guys would like to make a PR, we will gladly accept it.

Reply:
Yes know that Mariano :) Not asking you to do it in the reference pool, just some guidance is all.

We are not all mathematicians not blockchain experts nor all the intrinsic details of chia protocol, though I totally understand its not yours nor Chia problem ;)

Thanks anyway